Source/FFmpegMediaSource.cpp

```diff
@@ -190,7 +190,9 @@ namespace winrt::FFmpegInteropX::implementation
 
         if (SUCCEEDED(hr))
         {
-            hr = InitFFmpegContext();
+            hr = InitFFmpegContext();
+
+            CreateMediaStreamSource();
```

Comment:
Important.

CreateFrom* variants were **not** initializing the MSS member, leading to null pointer exceptions when trying to retrieve it.